### DISCUSSION TITLE
Tycho 4 at EclipseCon 2023

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-tycho/tycho#2640] [Announcements]
There will be a talk about [Tycho 4](https://github.com/eclipse-tycho/tycho/discussions/2455) at the [EclipseCon 2023](https://www.eclipsecon.org/2023), looking forward to see you there!

> If you have ever wondered how you can simplify your build or wanted to know how Tycho works under the hood, this is the right talk for you!

https://www.eclipsecon.org/2023/sessions/inside-tycho-4-build-bundles-better

**Comment by hd42** [comment]
Are the slides or maybe even a recording available?

**Reply by laeubi** [reply]
It usually takes a few weeks until the recordings are published, I'll try to post a link here then.

**Reply by dhendriks** [reply]
This one? https://www.youtube.com/watch?v=FIhVm_JfEBk
